x-pack/plugins/uptime/server/lib/alerts/status_check.ts

```diff
@@ -287,26 +304,51 @@ export const statusCheckAlertFactory: UptimeAlertTypeFactory<ActionGroupIds> = (
       };
 
       let downMonitorsByLocation: GetMonitorStatusResult[] = [];
+      let upMonitorsByLocation: GetMonitorStatusResult[] = [];
 
       // if oldVersionTimeRange present means it's 7.7 format and
       // after that shouldCheckStatus should be explicitly false
       if (!(!oldVersionTimeRange && shouldCheckStatus === false)) {
-        downMonitorsByLocation = await libs.requests.getMonitorStatus({
+        const queryConfig = {
           uptimeEsClient,
           timerange,
           numTimes,
           locations: [],
           filters: filterString,
-        });
+        };
+        downMonitorsByLocation = (await libs.requests.getMonitorStatus(queryConfig)) || [];
+        upMonitorsByLocation =
+          (await libs.requests.getMonitorStatus({
+            ...queryConfig,
+            status: 'up',
+          })) || [];
       }
+      const alertInstanceIds = new Set<string>();
```

Comment:
This is not supposed o be here. Remnant from a different interpretation I tried. Removing.